### PR TITLE
web-ui: let operator set the registry and tag

### PR DESCRIPTION
**Pull request by mareklibra**

### CODE REVIEW

File: pkg/controller/hyperconverged/hyperconverged_controller.go [hunk 1]
```diff
@@ -416,13 +415,13 @@ func newKWebUIForCR(cr *hcov1alpha1.HyperConverged) *kwebuis.KWebUI {
 			Labels: labels,
 		},
 		Spec: kwebuis.KWebUISpec{
-			Version:                         "latest",                             // TODO: image tag name, use Version ; https://github.com/kubevirt/hyperconverged-cluster-operator/pull/22/files
-			RegistryUrl:                     "",                                   // TODO: use ContainerRegistry  ; https://github.com/kubevirt/hyperconverged-cluster-operator/pull/22/files
-			RegistryNamespace:               "",                                   // keep blank, already in ContainerRegistry
+			Version:                         "",                                   // will be equal to the web-ui-operator version (image tag)
```
Comment (djzager) [inline pkg/controller/hyperconverged/hyperconverged_controller.go]: what do you think about not setting these at all and just having

```
kwebuis.KWebUISpec{
    OpenshiftMasterDefaultSubdomain: ...,
    PublicMasterHostname: ...,
}
```

Comment (rthallisey) [reply pkg/controller/hyperconverged/hyperconverged_controller.go]: You can set Version, Registry, ect.. in the web-ui operator if they are not set on the web-ui CR.  Example in [CDI](https://github.com/mhenriks/containerized-data-importer/blob/master/pkg/operator/controller/controller.go#L498).

Comment (mareklibra) [reply pkg/controller/hyperconverged/hyperconverged_controller.go]: Empty strings removed from here.
The web-ui-operator sets missing values based on ENV variables - https://github.com/kubevirt/web-ui-operator/pull/62/files

**Comment by mareklibra** [comment]
For downstream, the CSV must provide proper ENV variables for the web-ui-operator

**Comment by mareklibra** [comment]
@rthallisey , shall be manifest regenerated within this PR or independently, please?

**Comment by mareklibra** [comment]
Rebased.

```
$ ./hack/build-manifests.sh 
# github.com/kubevirt/hyperconverged-cluster-operator/vendor/kubevirt.io/kubevirt/pkg/kubecli
../../vendor/kubevirt.io/kubevirt/pkg/kubecli/kubevirt_test_utils.go:27:2: cannot use MockKubevirtClientInstance (type *MockKubevirtClient) as type KubevirtClient in return argument:
	*MockKubevirtClient does not implement KubevirtClient (missing Auditregistration method)
```

Any idea how to fix it, please?

**Comment by tiraboschi** [comment]
> Any idea how to fix it, please?

```
--- a/vendor/kubevirt.io/kubevirt/pkg/kubecli/kubevirt_test_utils.go
+++ b/vendor/kubevirt.io/kubevirt/pkg/kubecli/kubevirt_test_utils.go
@@ -24,7 +24,7 @@ var MockKubevirtClientInstance *MockKubevirtClient

 // GetMockKubevirtClientFromClientConfig is an entry point for testing, could be used to override GetKubevirtClientFromClientConfig
 func GetMockKubevirtClientFromClientConfig(cmdConfig clientcmd.ClientConfig) (KubevirtClient, error) {
-       return MockKubevirtClientInstance, nil
+       return nil, nil
 }
```

**Comment by mareklibra** [comment]
manifests are updated (last commit, atm)

**Pull request merged**